examples/md-flexible/src/configuration/objects/CubeClosestPacked.h

```diff
@@ -69,17 +103,20 @@ class CubeClosestPacked : public Object {
    */
   void generate(std::vector<ParticleType> &particles) const override {
     ParticleType particle = getDummyParticle(particles.size());
-
+    // Spacing in y direction when only moving 60° on the unit circle. Or the height in an equilateral triangle.
     const double spacingRow = _particleSpacing * sqrt(3. / 4.);
+    // Spacing in z direction. Height in an equilateral tetraeder.
     const double spacingLayer = _particleSpacing * sqrt(2. / 3.);
+    // Shorter part of the bisectrix when split at the intersection of all bisectrices.
     const double xOffset = _particleSpacing * 1. / 2.;
+    // Shorter part of the bisectrix when split at the intersection of all bisectrices.
```

Comment:
If I see this correctly, you compute the same values in `getParticlesTotal()`. Probably makes sense to promote these values to class members and calculate them in the constructor so we avoid the code duplication of the calculation.

Reply:
I agree, makes the methods look cleaner as well.